src/bindings/js/node/scripts/lib/utils.js

```diff
@@ -57,18 +57,15 @@ async function checkIfPathExists(path) {
  *
  * @function downloadFile
  * @param {string} url - The file URL.
- * @param {string} filename - The filename of result file.
- * @param {string} destination - The destination path of result file.
+ * @param {string} filePath - Path to downloaded file.
  * @param {string} [proxy=null] - (Optional) The proxy URL.
  * @returns {Promise<string>} - Path to downloaded file.
  */
-function downloadFile(url, destination, filename, proxy = null) {
-  console.log(`Downloading file by link: ${url} to ${destination}`
-    + `with filename: ${filename}`);
+function downloadFile(url, filePath, proxy = null) {
```

Comment:
This helper uses here: https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/js/node/scripts/lib/binary-manager.js#L97

Reply:
Yes, I've reverted the past behavior for `downloadFile`